src/main/webapp/WEB-INF/views/product_all.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ include file="header.jsp" %>
+
+<div class="container">    
+  <div class="row">
+  <c:forEach items="${products}" var="product">
+    <div class="col-sm-4">
+      <div class="panel panel-primary">
+        <div class="panel-heading"> ${product.name} </div>
+        <div class="panel-body">
+        <p>Price : ${product.price}  </p>
+		<p>Web Store Owner :  ${product.storeOwner} </p>  
+        </div>
+        <div class="panel-footer"> <button type="button" class="btn btn-primary btn-md"
+           onClick="location.href='/products/${product.id} ' ">Review</button></div>
+      </div>
+    </div>
+    </c:forEach>
+  </div>
+</div><br>
```

Comment:
avoid line break tags where possible.  also, use the self-closing syntax `<br />`

Reply:
Updated